### PR TITLE
Bug/ch1634/the filter sumary buttons links themselves

### PR DESCRIPTION
**Pull request by SamMed1**
Pushed two commits for accessibility of the filter 'tags', and then to take them out of the `<table> `structure, for screenreader compatibility. Some css amends to recreate how the table structure presented the elements.

### CODE REVIEW

File: web/sites/default/settings.php [hunk 1]
```diff
@@ -815,3 +815,13 @@
 elseif (getenv('ELASTICSEARCH_URL')) {
   $config['elasticsearch_connector.cluster.live_cluster']['url'] = getenv('ELASTICSEARCH_URL');
 }
+$databases['default']['default'] = array (
```
Comment (lewisnyman) [inline web/sites/default/settings.php]: We have to be careful not to commit  changes to this file as it's also used by production. I can help you sort this out (you can move the changes into settings.local.php)

Comment (SamMed1) [reply web/sites/default/settings.php]: can we gitignore?

Comment (joesb) [reply web/sites/default/settings.php]: I can talk you through a better way of doing this, @SamMed1

Comment (joesb) [reply web/sites/default/settings.php]: Aah, what Lewis said (apologies, didn't read properly!) — things like DB connections should be in `settings.local.php`. See step 4 of https://github.com/cabinetoffice/national-leadership-centre-drupal#installation-and-configuration

Comment (SamMed1) [reply web/sites/default/settings.php]: ok no probs, I don't actually remember changing anything in this file which is strange, @joesb would this have changed during onboarding yesterday perhaps?

**Comment by SamMed1** [comment]
`settings.php` file reverted to master so no difference. Is that right?

**Comment by joesb** [comment]
Superseded by #121 which is now merged to master

**Pull request closed without merging**